ascon/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "ascon"
+version = "0.1.4"
+authors = ["quininer kel <quininer@live.com>"]
+description = "A implementation of ASCON authenticated encryption."
+repository = "https://github.com/quininer/ascon"
+license = "MIT"
```

Comment:
@quininer is it okay if we dual license this code as Apache 2.0+MIT? We generally license all projects under that combination

Reply:
Of course :)